packages/governance/src/votes/votes.cairo

```diff
@@ -82,6 +80,25 @@ pub mod VotesComponent {
         pub const INVALID_SIGNATURE: felt252 = 'Votes: invalid signature';
     }
 
+    /// A trait that must be implemented when integrating {VotesComponent} into a contract. It
+    /// offers a mechanism to retrieve the number of voting units for a given account at the current
+    /// time.
+    pub trait VotingUnitsTrait<TState> {
+        /// Returns the number of voting units for a given account. For ERC20, this is typically the
+        /// token balance. For ERC721, this is typically the number of tokens owned.
+        ///
+        /// WARNING: While any formula can be used as a measure of voting units, the internal vote
+        /// accounting of the contract may be compromised if voting units are transferred in any
```

Comment:
This is technically breaking since we're changing its path, no? I'd probably add it to the changelog since it's public. WDYT?